src/discriminators/__tests__/composeChildTC-test.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import { schemaComposer } from 'graphql-compose';
+import { getCharacterModels } from '../../__mocks__/characterModels';
+import { composeWithMongooseDiscriminators } from '../../composeWithMongooseDiscriminators';
+
+const { CharacterModel, PersonModel, DroidModel } = getCharacterModels('type');
+
+beforeAll(() => schemaComposer.clear());
+
+describe('composeChildTC ->', () => {
+  const CharacterDTC = composeWithMongooseDiscriminators(CharacterModel);
+  const PersonTC = CharacterDTC.discriminator(PersonModel);
+  const DroidTC = CharacterDTC.discriminator(DroidModel);
+
+  it('should set DInterface to childTC', () => {
+    expect(DroidTC.hasInterface(CharacterDTC.getDInterface())).toBeTruthy();
+    expect(PersonTC.hasInterface(CharacterDTC.getDInterface())).toBeTruthy();
+  });
+
+  it('should copy all baseFields from BaseDTC to ChildTCs', () => {
+    expect(DroidTC.getFieldNames()).toEqual(expect.arrayContaining(CharacterDTC.getFieldNames()));
+    expect(PersonTC.getFieldNames()).toEqual(expect.arrayContaining(CharacterDTC.getFieldNames()));
```

Comment:
Getting some flow errors, you might want to check please. 
From lines 22, 23 ...
Error:(22, 68) Cannot call `expect.arrayContaining` with `CharacterDTC.getFieldNames()` bound to `value` because string [1] is incompatible with mixed [2] in array element.